package/remarkable-stylus/package

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+# Copyright (c) 2021 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(remarkable-stylus)
+pkgver=0.0.3-1
+pkgdesc="hack to get the button of a lamy pen working"
+timestamp=2020-11-19T20:07:29Z
+maintainer="Eeems <eeems@eeems.email>"
+license=LGPL-3.0
+url=https://github.com/ddvk/remarkable-stylus
+section="admin"
+installdepends=(display "tarnish=$pkgver")
```

Comment:
what's this about?

Reply:
Leftover from the package I copied in. That's part of why this is still a draft :P